### PR TITLE
Update usb-overdrive to 3.3

### PR DESCRIPTION
**Pull request by AlexandreBouffard**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/usb-overdrive.rb [hunk 1]
```diff
@@ -2,13 +2,15 @@
...
     version '3.2'
     sha256 '5be1d46596f1e98e29a5634655f36fc3070ac08542c36cef77d8f516c883f6d7'
+  else
+    version '3.3'
+    sha256 'aae9c1e92806090639ce50132fd214b0986dfa9a2687548f223575803ba0d28b'
   end
 
-  # senlick.com was verified as official when first introduced to the cask
-  url "http://www.senlick.com/macsw/USB-Overdrive-#{version.no_dots}.dmg"
+  url "http://www.usboverdrive.com/download/USB-Overdrive-#{version.no_dots}.dmg"
```
Comment (reitermarkus) [inline Casks/usb-overdrive.rb]: Do the old versions still work with this URL?

File: Casks/usb-overdrive.rb [hunk 2]
```diff
@@ -2,13 +2,15 @@
   if MacOS.version < :mountain_lion
     version '3.0.2'
     sha256 'd6bed831521a5ab2b82d7eede7be46953e1c010cc6e02ff11f3dd2532d7b0b13'
-  else
+  elsif MacOS.version < :mavericks
```
Comment (vitorgalvao) [inline Casks/usb-overdrive.rb]: Make it `<= :mountain_lion`. We prefer these to be inclusive.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/usb-overdrive.rb (hunk 1 above)
Comment (AlexandreBouffard) [inline Casks/usb-overdrive.rb]: Yes tested all 3 URLs.

File: Casks/usb-overdrive.rb (hunk 2 above)
Comment (AlexandreBouffard) [inline Casks/usb-overdrive.rb]: Done.

**Comment by reitermarkus** [comment]
Thanks!

**Pull request merged**